src/Contracts/Patternable.php

```diff
@@ -37,6 +37,7 @@ public function getPattern();
     public function setPattern(
         $prefix    = Filesystem::PATTERN_PREFIX,
         $date      = Filesystem::PATTERN_DATE,
+        $suffix    = Filesystem::PATTERN_SUFFIX,
         $extension = Filesystem::PATTERN_EXTENSION
     );
-}
+}
```

Comment:
don't remove newline, please.

I think reason like: ["No newline at end of file" compiler warning](https://stackoverflow.com/questions/72271/no-newline-at-end-of-file-compiler-warning) or Unix env.